lib/AST/ASTContext.cpp

```diff
@@ -1487,7 +1505,7 @@ ASTContext::associateInfixOperators(PrecedenceGroupDecl *left,
 }
 
 // Find library intrinsic function.
-static FuncDecl *findLibraryFunction(const ASTContext &ctx, FuncDecl *&cache, 
+static FuncDecl *findLibraryFunction(const ASTContext &ctx, FuncDecl *&cache,
```

Comment:
Discussed internally and this should rather be called `findStdlibFunction` IMHO.

Will do renames if we want to in separate PR

Reply:
rdar://78800086